src/FluxCalculator.jl

```diff
@@ -61,18 +61,18 @@ function turbulent_fluxes!(csf, model_sims, thermo_params)
 
     # Reset the coupler fields will compute. We need to do this because we will compute
     # area-weighted averages
-    csf.F_turb_ρτxz .*= FT(0)
-    csf.F_turb_ρτyz .*= FT(0)
-    csf.F_lh .*= FT(0)
-    csf.F_sh .*= FT(0)
-    csf.F_turb_moisture .*= FT(0)
-    csf.z0m_sfc .*= FT(0)
-    csf.z0b_sfc .*= FT(0)
-    csf.beta .*= FT(0)
-    csf.q_sfc .*= FT(0)
-    csf.L_MO .*= FT(0)
-    csf.ustar .*= FT(0)
-    csf.buoyancy_flux .*= FT(0)
+    fill!(csf.F_turb_ρτxz, FT(0))
+    fill!(csf.F_turb_ρτyz, FT(0))
+    fill!(csf.F_lh, FT(0)) 
+    fill!(csf.F_sh, FT(0)) 
+    fill!(csf.F_turb_moisture, FT(0))
+    fill!(csf.z0m_sfc, FT(0))
+    fill!(csf.z0b_sfc, FT(0)) 
+    fill!(csf.beta, FT(0)) 
+    fill!(csf.q_sfc, FT(0)) 
+    fill!(csf.L_MO, FT(0)) 
+    fill!(csf.ustar, FT(0)) 
+    fill!(csf.buoyancy_flux, FT(0))
```

Comment:
You could write
```suggestion
for p in (:F_turb_.....)
fill!(getproperty(csf, p), 0)
end
```
and maybe 0 works (instead of FT(0))